bin/node-template/node/Cargo.toml

```diff
@@ -20,52 +20,52 @@ name = "node-template"
 clap = { version = "4.0.9", features = ["derive"] }
 futures = { version = "0.3.21", features = ["thread-pool"]}
 
-sc-cli = { version = "0.10.0-dev", path = "../../../client/cli" }
-sp-core = { version = "7.0.0", path = "../../../primitives/core" }
-sc-executor = { version = "0.10.0-dev", path = "../../../client/executor" }
-sc-service = { version = "0.10.0-dev", path = "../../../client/service" }
-sc-telemetry = { version = "4.0.0-dev", path = "../../../client/telemetry" }
-sc-keystore = { version = "4.0.0-dev", path = "../../../client/keystore" }
-sc-transaction-pool = { version = "4.0.0-dev", path = "../../../client/transaction-pool" }
-sc-transaction-pool-api = { version = "4.0.0-dev", path = "../../../client/transaction-pool/api" }
-sc-consensus-aura = { version = "0.10.0-dev", path = "../../../client/consensus/aura" }
-sp-consensus-aura = { version = "0.10.0-dev", path = "../../../primitives/consensus/aura" }
-sp-consensus = { version = "0.10.0-dev", path = "../../../primitives/consensus/common" }
-sc-consensus = { version = "0.10.0-dev", path = "../../../client/consensus/common" }
-sc-finality-grandpa = { version = "0.10.0-dev", path = "../../../client/finality-grandpa" }
-sp-finality-grandpa = { version = "4.0.0-dev", path = "../../../primitives/finality-grandpa" }
-sc-client-api = { version = "4.0.0-dev", path = "../../../client/api" }
-sp-runtime = { version = "7.0.0", path = "../../../primitives/runtime" }
-sp-io = { version = "7.0.0", path = "../../../primitives/io" }
-sp-timestamp = { version = "4.0.0-dev", path = "../../../primitives/timestamp" }
-sp-inherents = { version = "4.0.0-dev", path = "../../../primitives/inherents" }
-sp-keyring = { version = "7.0.0", path = "../../../primitives/keyring" }
-frame-system = { version = "4.0.0-dev", path = "../../../frame/system" }
-pallet-transaction-payment = { version = "4.0.0-dev", default-features = false, path = "../../../frame/transaction-payment" }
+sc-cli = { version = "0.12.0", path = "../../../client/cli" }
+sp-core = { version = "9.0.0", path = "../../../primitives/core" }
```

Comment:
Why did it bump `sp-core` from `7.0.0` to `9.0.0`?

Reply:
8.0.0 was released when I ran the job manually before. This is the first (automated) run in which the PR was actually opened, so it looks like it "skipped" 8.0.0, but it didn't. See https://crates.io/crates/sp-core/versions.